src/web/screens/emailContacts.tsx

```diff
@@ -75,7 +75,7 @@ export function BusinessContacts() {
   const handleAddEmailContact = async (formData: BusinessContactForm) => {
     try {
       const response = await AddEmailContact(formData);
-      if (response.statusText === 'OK') {
+      if (response.status === 200) {
```

Comment:
nit: I wonder if we should change the route to return 201

Reply:
i now return and look for a 201